backend/tests/test_auth_api.py

```
@@ -0,0 +1,27 @@
+from user_management.models import User
+def test_register(client, db_session):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_register`

Add return type annotation: `None`

(ANN201)